internal/trace/sql.go

```diff
@@ -0,0 +1,26 @@
+package trace
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "modernc.org/sqlite"
```

Comment:
This sqlite package is a pure Go implementation (does not require CGO). I've used it before and it's quite good.